src/frontend/src/handler/create_table.rs

```diff
@@ -488,6 +464,23 @@ pub fn bind_pk_and_row_id_on_relation(
         )))?;
     }
 
+    // Check if impure generated columns are part of the primary key.
+    for column in &columns {
```

Comment:
This step used to be in `bind_sql_column_constraints`, now moved into `bind_pk_and_row_id_on_relation`.